demisto_sdk/__main__.py

```diff
@@ -1525,15 +1526,14 @@ def upload(ctx, **kwargs):
 @click.pass_context
 @logging_setup_decorator
 def download(ctx, **kwargs):
-    """Download custom content from Demisto instance.
-    DEMISTO_BASE_URL environment variable should contain the Demisto server base URL.
-    DEMISTO_API_KEY environment variable should contain a valid Demisto API Key.
+    """Download custom content from a Cortex XSOAR / XSIAM instance.
```

Comment:
Just tested, seems to work fine on XSIAM.

The endpoint we use to download the custom content bundle (`/content/bundle`) seems to exist and work the same on XSIAM.

Reply:
From my testings, yes, it returns the same `tar.gz` content bundle as XSOAR (which is the only API call we use when downloading custom content).
There could be differences however for downloading system content (which is not relevant for this function), since there are different endpoints in use for that.